### PR TITLE
SOBACK-115 & SOBACK-116 + minor UI improvement

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Added error message when we reach 500 or more images

Added error when backend doesn't reply properly

Made it so when you press enter in any of the longitude/latitude/radius/query input fields, we fire a search. Was tedious work having to press the button each time >.<

@thomasegense for review of functionality
@jorntx for code review (not much to look at tho, should be quick)

### CODE REVIEW

**Review by thomasegense — APPROVED** [review]
OK

File: src/js/src/store/modules/notifier.store.js [hunk 1]
```diff
@@ -24,7 +24,12 @@
   const mutations = {
    
     setNotification(state, notification) {
-      state.notifications.push(notification)
+      if(notification === undefined) {
+        state.notifications.push(notification)
+      }
+      else {
+        state.notifications.push(notification)
+      }
```
Comment (jorntx) [inline src/js/src/store/modules/notifier.store.js:32]: What is the purpose if this? 🤔

**Review by jorntx — APPROVED** [review]
Works as it should - good work!

Only found a small code thing I need an explanation for.

**Review by jorntx — APPROVED** [review]
Let's go!

**Pull request merged**